src/fmu/sumo/uploader/_caseondisk.py

```diff
@@ -129,7 +129,8 @@ def _get_sumo_parent_id(self):
         Call sumo, check if the case is already there. Use fmu_case_uuid for this."""
 
         query = f"fmu.case.uuid:{self.fmu_case_uuid}"
-        search_results = self.sumo_connection.api.searchroot(query, search_size=2)
+        path = f"/searchroot?$query={query}&$from={0}&$size={2}"
+        search_results = self.sumo_connection.api.get(path)
```

Comment:
In `SumoClient.get` you dont have to put the query parameters in the path. `kwargs` will be treated like query parameters so you can do stuff like this:

```python
sumo.get("/searchroot",  query=f"fmu.case.uuid:{self.fmu_case_uuid}", size=2)
```

Reply:
This is applied in the latest commit. Please check :)